### ISSUE TITLE
create config.py on basic folder

### ISSUE THREAD

**Issue by fernandascovino** [JoaoCarabetta/project-templates#2]
To import direct from the notebook/script.

Content: 

```
# FILE STORAGE
import sys
import os
from pathlib import Path 
current_path = Path().resolve()
abs_path = str(current_path.parent)
sys.path.append(abs_path)

RAW_PATH = current_path / 'data' / 'raw'
TREAT_PATH = current_path / 'data' / 'treated'
OUTPUT_PATH = current_path / 'data' / 'output'
```

**Comment by fernandascovino** [comment]
And the notebooks/scripts need this:

```
# Import config
import sys
sys.path.insert(0, '../')
from config import RAW_PATH, TREAT_PATH, OUTPUT_PATH
```

P.S.: I'm using a treated folder, I think is good to have one (when you just filter ou clean the raw data, it's not the output yet)

**Comment by JoaoCarabetta** [comment]
Was this closed by the last PR?

**Comment by fernandascovino** [comment]
If you had read the PR, you would know... the description is "Closes #5 and #2"

**Comment by fernandascovino** [comment]
It didn't work?
